css/menu.css

```
@@ -256,6 +256,10 @@ body.has-menu-left.has-push-menu .fl-viewport-header {
   box-shadow: 4px 0 5px rgba(0,0,0,0.2);
 }
 
+.fl-menu-push-in-left .fl-close-menu.focus-outline {
```

Why is the `.focus-outline` part necessary?

```suggestion
.fl-menu-push-in-left .fl-close-menu {
```